### PR TITLE
Microsoft Authentication - National Endpoints

### PR DESCRIPTION
**Pull request by darkushin**
## Status
In Progress

## Related Issues
related: https://github.com/demisto/etc/issues/30917

## Description
Add to the Microsoft Integrations documentation an explanation about using National Cloud.

### CODE REVIEW

**Comment by darkushin** [comment]
Waiting until we merge the changes to the integrations and will then release the docs update.

**Comment by ShirleyDenkberg** [comment]
@darkushin I was unable to commit my changes (which were very minor). 
Just replace lines 66-72 with the following:
#### Using National Cloud
Some of the Cortex XSOAR-Microsoft integrations support the deployment of national clouds through the self-deployed
 authorization flow. For more information about Microsoft National Clouds, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/graph/deployments).
 Currently, the following integration supports national clouds:
 * O365 Outlook Mail (Using Graph API)
In order to use a national cloud, change the *URL* parameter to the corresponding address of the national cloud you are using. For example, see [O365 Outlook Mail (Using Graph API) - National Clouds](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-mail#using-national-cloud), for the supported cloud endpoints in this integration.

**Comment by ShirleyDenkberg** [comment]
@bakatzir I was unable to commit my changes so I sent them to @darkushin to make.  Other than that, doc review completed.

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**
